vetiver/meta.py

```diff
@@ -25,9 +27,10 @@ def from_dict(cls, metadata, pip_name=None) -> "VetiverMeta":
         version = metadata.get("version", None)
         url = metadata.get("url", None)
         required_pkgs = metadata.get("required_pkgs", [])
+        python_version = metadata.get("python_version", sys.version)
```

Comment:
```suggestion
        python_version = metadata.get("python_version", sys.version_info)
```

Later on, I think we want to be able to quickly compare the version of Python that the model was originally written with  the version of Python that the user has locally (if model is read in from some other location). Using `version_info` for a tuple rather than a string will probably be more straightforward for this comparison.

Reply:
Ah, I see that! I don't believe there is a cleaner approach, but will keep that in mind 👀